src/test/java/net/datafaker/providers/entertainment/OscarMovieTest.java

```diff
@@ -33,4 +41,29 @@ void character() {
     void releaseDate() {
         assertThat(oscarMovie.releaseDate()).matches("[A-Za-z,0-9 ]+");
     }
+
+    /**
+     * Test for <a href="https://github.com/datafaker-net/datafaker/issues/741">issue741</a>
+     */
+    @ParameterizedTest
+    @MethodSource("argsProvider")
+    void issue741(Function<OscarMovie, String> f) {
+        final OscarMovie oscarMovie = faker.oscarMovie();
+        assertThat(
+            faker.stream(() -> f.apply(oscarMovie)).len(10).build()
+                .<Stream<?>>get().collect(Collectors.toSet()))
+            .hasSizeGreaterThan(1);
+    }
```

Comment:
This doesn't really test that the values aren't consistent, just that something is generated. (Or I'm missing something?)

Reply:
the problem was that it generated same values for a series of invocations like
 ```java
     Faker faker = new Faker();

    String movie1 = faker.oscarMovie().movieName();
    String movie2 = faker.oscarMovie().movieName();
    String movie3 = faker.oscarMovie().movieName();
    ```
before that fix it generates same values for all 3 vars.

So the test checks that the generated value are not same after 10 times of invocations

Reply:
Sorry missed it was collecting to set which would mean unique entries. Tks